libs/beacon/src/trusted_dealer_beacon_service.cpp

```diff
@@ -36,7 +36,7 @@ void TrustedDealerSetupService::StartNewCabinet(
     BlockEntropy const &prev_entropy, DkgOutput const &output,
     std::pair<SharedNotarisationManager, CabinetNotarisationKeys> notarisation_keys)
 {
-  uint64_t round_end = round_start + aeon_period_ - 1;
+  uint64_t round_end = round_start + aeon_period_;
```

Comment:
This caused the beacon to trigger the new out of sync functionality erronerously

Reply:
From what I understood of the set up the round start and round end variables are inclusive, i.e. the cabinet is in charge of the end points as well. If this is the case then the -1 one is consistent with the total number of rounds in the aeon being aeon_period. 

See for example how the values are set in consensus.cpp:
```
 cabinet_creator_->StartNewCabinet(cabinet_member_list, threshold,
                                        current_block_.block_number + 1,
                                        current_block_.block_number + aeon_period_,
                                        last_block_time + block_interval, current.block_entropy);
```

Reply:
I think there is slight incongruity with its actual use which led to my confusion. The variable is indeed inclusive, here is its use in consensus.cpp:

```
      // Safe to call this multiple times
      cabinet_creator_->StartNewCabinet(cabinet_member_list, threshold,
                                        current_block_.block_number + 1,
                                        current_block_.block_number + aeon_period_,
                                        last_block_time + block_interval, current.block_entropy);
```

the plus 1 is for the start since conceptually you already have genesis, whereas in the test it starts at 0. I have reverted the 'fix' and we can see if the CI passes now :)